### ISSUE TITLE
Integrate AuthorizedVoters into VotingController

### ISSUE THREAD

**Issue by jwinzer** [OpenSlides/openslides-voting#51]
We should consider integrating fields 'type' and 'authorized_voters' into VotingController and eliminating AuthorizedVoters.

@FinnStutzenstein: What are your thoughts?

**Comment by FinnStutzenstein** [comment]
I' wasnt happy about creating an extra model, too. But this had two reasons:
- Permissions. Not for the VoteCollector, but for the other modes, the client must know, whether his user is allowed to vote. I didn't want to give read access to the voting-controller, because a user may not see how many votes come in, ...
- Simpler use in the client. Because it does not have the `voting_target` as an integer but a real foreign key, the handling is much simpler than the voting-controller.

We may combine there and make different serializers. And yes, these models are somehow redundent. The `AuthorizedVoters` model was desinged to just give structured information to the client, so if we watch out for the few places where these information changes (mainly in `start`, `stop`), this should work.

So this needs to be dicided:
Does we want some rework or let this as it is. I can do this, if you want. Currently I have no strong opinion there (I had when I created the model, but after the development I saw that this might be obsolete)
What are your reasons your considering to merge the model?
